src/apps/mesoscale-explorer/ui/entities.tsx

```diff
@@ -777,7 +808,34 @@ export class EntityNode extends Node<{}, { action?: 'color' | 'clip', isDisabled
         if (e.ctrlKey) {
             this.toggleSelect(e);
         } else {
-            this.center(e);
+            const d = getEntityDescription(this.plugin, this.cell);
+            if (this.cell?.obj?.data.sourceData.state.models.length !== 0) {
+                const repr = this.cell?.obj?.data.repr;
+                if (repr) {
```

Comment:
When you click on a given leaf node in the right panel tree, it will center the camera in a sequential manner iterating other the instances. I found it usefull.